Lib/test/test_email/test_email.py

```diff
@@ -31,6 +31,7 @@
 from email.mime.message import MIMEMessage
 from email.mime.multipart import MIMEMultipart
 from email.mime.nonmultipart import MIMENonMultipart
+from email.headerregistry import HeaderRegistry
```

Comment:
Please sort these imports alphabetically.

Reply:
@warsaw Could you please be a little bit more specific about which imports to sort? All of the ones starting with `from email.`?

Reply:
Ah no, sorry about that.  There's some history with this as an external library, but basically keep the format in the file where `from email.foo` imports come first in that block, but sorted alphabetically, followed by `from email import` stuff.  Really it just comes down to putting the new `from email.headerregistry` line right after the `from email.header` line (and before the `from email.parser` line).